common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -425,6 +432,21 @@ def test_discussion_blackout_period(self):
         self.assertFalse(self.thread_page.is_comment_deletable("comment1"))
         self.assertFalse(self.thread_page.is_comment_deletable("comment2"))
 
+    def test_dual_discussion_module(self):
+        self.discussion_page.wait_for_page()
+        self.additional_discussion_page.wait_for_page()
+        self.discussion_page.click_new_post_button('.new-post-btn')
+        with self.discussion_page.handle_alert():
+            self.discussion_page.click_cancel_new_post('.cancel')
+        self.additional_discussion_page.click_new_post_button('.new-post-btn')
```

Comment:
Why does click_new_post_button need to take the class name as an input? Isn't it always the same?

Same question for cancel.